cps/db.py

```diff
@@ -973,7 +974,19 @@ def get_cc_columns(self, config, filter_config_custom_read=False):
     def get_search_results(self, term, config, offset=None, order=None, limit=None, *join):
         order = order[0] if order else [Books.sort]
         pagination = None
-        result = self.search_query(term, config, *join).order_by(*order).all()
+        
+        # search also through the subtitles (for videos)
+        searcher = lb_search.CaptionSearcher(XKLB_DB_FILE)
+        other_terms = searcher.get_search_terms(term)
+        # lb_search.get_search_terms returns a list of video titles, "term" parameter is expected to be a book/video title
+        term = [term] + other_terms
```

Comment:
Per discussions, change variable term to a name reflective of what it's doing (something like termList or titleList ?). 

Separate search query from results/video_titles into two variables.

Reply:
term variable is from upstream. it is used throughout db.py. The PR was refactored to output ids instead of video titles.